### ISSUE TITLE
array dimension not consistent error

### ISSUE THREAD

**Issue by qigangliu** [PiscesDream/CycleGAN-keras#1]
this project is great, i just search for a CycleGAN implemented in  keras, when i run  the  demo, i encounter a error "expected input_3 to have shape (None, 3, 128, 128) but got array with shape (1, 128, 128, 3)" accured  in "fake_A_pool.extend(self.BtoA.predict(real_B))" , it seems there are some problems with the array dimensions  conversion, can you confirm it, thank you very much.

**Comment by Shaofanl** [comment]
Indeed. There is a difference between TF and Theano in dimension. I think I used `"image_data_format": "channel_last"` in `$HOME/.keras/keras.json` when I wrote this repository. Since they stop to update Theano anymore, I recommend you to change everything to TF-like yourself.
